README.md

```diff
@@ -847,154 +597,125 @@ are defined with a single style rule. You shouldn't be using these a lot.
 These styles should not be changed, since they will have side-effects in
 unpredictable places.
 
+
 <a name="ns--namespace"></a>
 #### ns- Namespace.
 
 `ns-newcsslib` is used to insulate pages or sections so that styles from this library apply there, but not anywhere else. Any other namespace classes should be used extremely sparingly. They are meant to be temporary.
 
 
-<a name="-a-hack"></a>
-#### _ a Hack.
-
-Used sparingly for cross-browser compatibility. Example:
-
-```CSS
-._media {zoom:1} /* Gives element 'has-layout' in IE */
-```
-
 <a name="js--javascript"></a>
 #### js- Javascript.
 
 Used for scripting hooks only. Scripts should not make reference to any other classes.
 
+
 <a name="test--test"></a>
 #### test- Test.
 
 Used for automated testing hooks only. Test scripts should not make reference to any other classes.
 
-<a name="s--scope"></a>
-#### s- Scope.
-
-Resets all the styles for a given context, by brute force. Would only be used in very specific scenarios. We're not using this currently.
-
-
-
-
 
+<a name="use-bem-to-reflect-component-structure"></a>
+### Use BEM to reflect component structure
 
+BEM convention (Block, Element, Modifier) uses different delimiters to make it easier to understand the role of an element.
```

Comment:
We should include a link to a good primer though. This style guide should be for writers of CSS and those trying to read it/or are curious about it, so it can and should get into the technical stuff. I just don't think we need to repeat what others have written better when we can just link to that article instead. The live style guide is for consumers of music. Any docs there should relate only to the use of the library and not require any specialized CSS knowledge.